specs/_features/eip7251/beacon-chain.md

```diff
@@ -87,6 +90,12 @@ See [a modest proposal](https://notes.ethereum.org/@mikeneuder/increase-maxeb),
 
 The following values are (non-configurable) constants used throughout the specification.
 
+### Misc
+
+| Name | Value |
+| - | - |
+| `FULL_EXIT_REQUEST_AMOUNT` | `uint64(2**64 - 1)` |
```

Comment:
I think this is fine, but on the EL side we should make sure that the contract is set up so that full exits do not become more expensive, i.e., the `amount` parameter should just be ignored for a full exit

Reply:
This is a good point!

I did a bit more thinking and realized that the most optimal way would be using `0` at the smart contract level. And then changing `0` to `0xff..` on whatever downstream layer does seem odd to me. So probably the best way would be using `uint64(0)` on CL as well